modules/op-release-notes-1-12.adoc

```diff
@@ -0,0 +1,192 @@
+// This module is included in the following assemblies:
+// * about/op-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="op-release-notes-1-12_{context}"]
+= Release notes for {pipelines-title} General Availability 1.12
+
+With this update, {pipelines-title} General Availability (GA) 1.12 is available on {OCP} 4.12 and later versions.
+
+[id="new-features-1-12_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.12:
+
+[NOTE]
+====
+Before upgrading to the {pipelines-title} Operator 1.12, ensure that you have at least installed the {OCP} 4.12.19 or 4.13.1 version on your cluster.
+====
+
+[id="pipelines-new-features-1-12_{context}"]
+=== Pipelines
+
+* With this update, the web console includes a new gauge metric for pipeline runs and task runs. This metric indicates whether the underlying pods are being throttled by {OCP} because of either ResourceQuota policies defined in the namespace or because of resource constraints on the underlying node.
+
+* With this update, the new `set-security-context` feature flag is set to true by default, in order to enable task runs and pipeline runs to run in namespaces with restricted pod security admission policies.
+
+* With this update, the `enable-api-fields` flag is set to `beta` by default. You can use all features that are marked as “beta” in the code without further changes.
+
+* With this update, the `results.tekton.dev/\*` and `chains.tekton.dev/*` reserved annotations are not passed from the pipeline run to the task runs that it creates.
+
+* Before this update, CSI volumes and projected volumes were not enabled by default. With this update, you can use CSI volumes and projected volumes in your pipelines without changing any configuration fields.
+
+* With this update, the isolated workspaces feature is enabled by default. With this feature, users can share a workspace with specified steps and sidecars without sharing it with the entire task run.
+
+[id="operator-new-features-1-12_{context}"]
+=== Operator
+
+* With this update, you can configure the default security context constraint (SCC) for the pods that {pipelines-shortname} creates for pipeline runs and task runs. You can set the SCC separately for different namespaces and also configure the maximum (least restrictive) SCC that can be set for any namespace.
+
+* With this update, a new `options:` heading is available under each component in the TektonConfig spec. You can use parameters under this headings to control settings for different components. In particular, you can use parameters under the `platforms.openshift.pipelinesAsCode.options.configMaps.pac-config-logging.data` spec to set logging levels for components of {pac}.
```

Comment:
"Spec" is the accepted term because it means the part of the YAML manifest of a CR that is under the `spec:` heading; I used it in other docs already, one of the senior writers, Ashling I think, gave it to me in review. The `options:` headings are plural because one is available under each component.